htdocs/bom/tpl/objectline_view.tpl.php

```diff
@@ -177,13 +177,19 @@
 	print '</td>';
 
 	// Work station
-	if (isModEnabled('workstation')) {
+	if (isModEnabled('workstation') && !empty($line->fk_default_workstation)) {
```

Comment:
If there is 2 lines, one with default workstation and one without,
we will get a <td> for one and not for the other.
This introduces surely a bug.

Reply:
Done